src/store/actions/createWallet.js

```diff
@@ -2,7 +2,21 @@ import { v4 as uuidv4 } from 'uuid'
 import { encrypt } from '../../utils/crypto'
 import buildConfig from '../../build.config'
 import { accountCreator, getNextAccountColor } from '@/utils/accounts'
-import { ChainId, chains, assets as cryptoassets } from '@liquality/cryptoassets'
+import { ChainId, chains, assets as cryptoassetsOrigin } from '@liquality/cryptoassets'
+
+// TO-DO: remove this after npm package is updated.
```

Comment:
I can update the crypto assets package for you let me know, or will need to wait for any other details related to this assets ?

BR

Reply:
@business-helper

Reply:
Yes, I think you can update the assets package,
though smart contracts are not deployed yet, you can add ZERO to the assets as the published version doesn't list ZERO, but I will use it in the development.
After smart contract deployment, we can update the assets again with real data.

Reply:
@business-helper can you please help me to get some ZERO in testnet ? to test the ui